doc/whats_new.rst

```diff
@@ -90,6 +90,10 @@ Bug fixes
      `#6497 <https://github.com/scikit-learn/scikit-learn/pull/6497>`_
      by `Sebastian Säger`_
 
+   - Fix bug svm's decision values when `decision_function_shape` is
+     `'ovr'` :class:`svm.SVC`.
```

Comment:
Please add that SVC's `decision_function` was incorrect for versions 0.17.0 through 0.18.0.